### PR TITLE
Hf rule0001 mod

### PR DESCRIPTION
**Pull request by hefischer**
Update rule 0001 to separate fact with different values/periods/units to independent messages, leaving comments on how to re-group messages if needed later.

### CODE REVIEW

**Comment by sagesmith-wf** [comment]
Failed Travis-ci

File: dqc_us_rules/dqc_us_0001.py [hunk 1]
```diff
@@ -202,18 +202,19 @@ def _run_member_checks(axis, axis_key, axis_config, relset, val, role,
...
+                            '{base_key}.{extension_key}'.format(
+                                base_key=_CODE_NAME,
+                                extension_key=axis_config[_RULE_INDEX_KEY]
+                            ),
+                            messages.get_message(_CODE_NAME, _UGT_FACT_KEY),
+                            axis=axis.label(),
+                            member=child.label(),
+                            modelObject=fact, #fact_list,
+                            ruleVersion=_RULE_VERSION,
+                            # errorCount=len(fact_list)
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Delete this line

File: dqc_us_rules/dqc_us_0001.py [hunk 2]
```diff
@@ -202,18 +202,19 @@ def _run_member_checks(axis, axis_key, axis_config, relset, val, role,
...
+                    for fact in fact_list: # remove for grouped messages
+                        val.modelXbrl.error(
+                            '{base_key}.{extension_key}'.format(
+                                base_key=_CODE_NAME,
+                                extension_key=axis_config[_RULE_INDEX_KEY]
+                            ),
+                            messages.get_message(_CODE_NAME, _UGT_FACT_KEY),
+                            axis=axis.label(),
+                            member=child.label(),
+                            modelObject=fact, #fact_list,
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Delete the comment

File: dqc_us_rules/dqc_us_0001.py [hunk 3]
```diff
@@ -269,17 +270,18 @@ def _run_extension_checks(axis, axis_key, axis_config, relset, val, role,
...
+                        for fact in fact_list: # remove for grouped messages
+                            val.modelXbrl.error(
+                                '{base_key}.{extension_key}'.format(
+                                    base_key=_CODE_NAME,
+                                    extension_key=axis_config[_RULE_INDEX_KEY]
+                                ),
+                                messages.get_message(_CODE_NAME, _EXT_FACT_KEY),
+                                axis=axis.label(),
+                                member=child.label(),
+                                modelObject=fact, # fact_list,
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Remove the comment

File: dqc_us_rules/dqc_us_0001.py [hunk 4]
```diff
@@ -269,17 +270,18 @@ def _run_extension_checks(axis, axis_key, axis_config, relset, val, role,
...
+                                ruleVersion=_RULE_VERSION,
+                                # errorCount=len(fact_list)
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Delete this line

File: dqc_us_rules/dqc_us_0001.py [hunk 5]
```diff
@@ -202,18 +202,19 @@ def _run_member_checks(axis, axis_key, axis_config, relset, val, role,
...
-                        ruleVersion=_RULE_VERSION,
-                        errorCount=len(fact_list)
-                    )
+                    for fact in fact_list: # remove for grouped messages
+                        val.modelXbrl.error(
+                            '{base_key}.{extension_key}'.format(
+                                base_key=_CODE_NAME,
+                                extension_key=axis_config[_RULE_INDEX_KEY]
+                            ),
+                            messages.get_message(_CODE_NAME, _UGT_FACT_KEY),
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Line too long suggestion:

```
message.get_message(
    _CODE_NAME, _UGT_FACT_KEY
),
```

File: dqc_us_rules/dqc_us_0001.py [hunk 6]
```diff
@@ -269,17 +270,18 @@ def _run_extension_checks(axis, axis_key, axis_config, relset, val, role,
...
+                                    base_key=_CODE_NAME,
+                                    extension_key=axis_config[_RULE_INDEX_KEY]
+                                ),
+                                messages.get_message(_CODE_NAME, _EXT_FACT_KEY),
+                                axis=axis.label(),
+                                member=child.label(),
+                                modelObject=fact, # fact_list,
+                                ruleVersion=_RULE_VERSION,
+                                # errorCount=len(fact_list)
                         )
```
Comment (sagesmith-wf) [inline dqc_us_rules/dqc_us_0001.py]: Line under-indented suggestion:

```
    ruleVersion=_RULE_VERSION,
)
```

**Comment by andrewperkins-wf** [comment]
Fixed in another PR because of miscommunication - closing

**Pull request closed without merging**